pkg/proxy/ipvs/proxier_test.go

```diff
@@ -49,6 +50,36 @@ type fakeIPGetter struct {
 	nodeIPs []net.IP
 }
 
+// expectVs is the expected ipvs rules with vs and rs
+// vsNum is the expected ipvs virtual server number
+// ip:port protocol is the expected ipvs vs info
+// rs is the real server of this expected vs
+type expectVs struct {
```

Comment:
Can you please add these structs to pkg/proxy/ipvs/testing/. Having them here clutters the test.

Reply:
Good point. I added a new util.go in `pkg/proxy/ipvs/testing/` to do that, if we want to enhance ipvs ut, those structs are very useful, I think. : )